sapi/src/lib.rs

```diff
@@ -45,87 +66,106 @@ pub struct SAPI {
 
 impl SAPI {
     /// initialize SAPI client API
-    pub fn new(
-        sapi_base_url: &str,
-        request_timeout: u64,
-        log: Logger,
-    ) -> Self {
+    pub fn new(sapi_base_url: &str, request_timeout: u64, log: Logger) -> Self {
         let client = Client::builder()
             .timeout(Duration::from_secs(request_timeout))
             .build()
             .unwrap();
-        let sapi = SAPI {
-                sapi_base_url: sapi_base_url.into(),
-                request_timeout: request_timeout.into(),
-                client,
-                log: log.clone(),
-        };
-        sapi
+        SAPI {
+            sapi_base_url: sapi_base_url.into(),
+            request_timeout,
+            client,
+            log: log.clone(),
+        }
     }
 
     /// Retrieve the "zone" configuration by zone UUID.
-    pub fn get_zone_config(
-        &self,
-        uuid: &str
-    ) -> Result<ZoneConfig, Box<dyn std::error::Error>> {
+    pub fn get_zone_config(&self, uuid: &str) -> Result<ZoneConfig, Box<dyn std::error::Error>> {
```

Comment:
Just making sure I understand: the API call here:

https://github.com/joyent/manta-buckets-mdapi/blob/master/schema-manager/src/main.rs#L54

Should not have to change, correct?

Reply:
This is just a formatting change.  Probably came in when I ran `cargo fmt`.  

Couple things to consider though.
* I didn't change the `ZoneConfig` format, but we probably should.
* rust-triton-clients is now versioned, and schema manager uses `v0.2.0`.  So even if this did change, keeping schema manager locked at that version would prevent any breakage.